docs/sources/setup-grafana/start-restart-grafana.md

```diff
@@ -0,0 +1,193 @@
+  - ../installation/restart-grafana/
+description: How to start the Grafana server
+title: Start the Grafana server
+menuTitle: Start Grafana
+weight: 300
+---
+
+# Start the Grafana server
+
+This topic includes instructions for starting the Grafana server. For certain configuration changes, you might have to restart the Grafana server for them to take effect.
+
+The following instructions start the `grafana-server` process as the `grafana` user, which was created during the package installation.
+
+If you installed with the APT repository or `.deb` package, then you can start the server using `systemd` or `init.d`. If you installed a binary `.tar.gz` file, then you execute the binary.
+
+## Linux
+
+Starting and restarting the Grafana server depends on whether your Linux system uses `systemd` or `init.d`.
+
+### Start the Grafana server with systemd
+
+Complete the following steps to start the Grafana server using systemd and verify that it is running:
+
+1. To start the service, run the following commands:
+
+   ```bash
+   sudo systemctl daemon-reload
+   sudo systemctl start grafana-server
+   sudo systemctl status grafana-server
+   ```
+
+1. To verify that the service is running, run the following command:
+
+   ```
+   sudo systemctl status grafana-server
+   ```
+
+### Configure the Grafana server to start at boot using systemd
+
+To configure the Grafana server to start at boot, run the following command:
+
+```bash
+sudo systemctl enable grafana-server.service
+```
+
+#### Serve Grafana on a port < 1024
+
+{{< docs/shared "systemd/bind-net-capabilities.md" >}}
+
+### Restart the Grafana server using systemd
+
+To restart the Grafana server, run the following commands:
+
+```bash
+sudo systemctl restart grafana-server
+```
+
+> **Note:** SUSE or OpenSUSE users might need to start the server with the systemd method, then use the init.d method to configure Grafana to start at boot.
+
+### Start the Grafana server using init.d
+
+Complete the following steps to start the Grafana server using init.d and verify that it is running:
+
+1. To start the Grafana server, run the following commands:
+
+   ```bash
+   sudo service grafana-server start
+   sudo service grafana-server status
+   ```
+
+1. To verify that the service is running, run the following command:
+
+   ```
+   sudo service grafana-server status
+   ```
+
+### Configure the Grafana server to start at boot using init.d
+
+To configure the Grafana server to start at boot, run the following command:
+
+```bash
+sudo update-rc.d grafana-server defaults
+```
+
+#### Restart the Grafana server using init.d
+
+To restart the Grafana server, run the following commands:
+
+```bash
+Need input
+```
+
+### Start the server using the binary
+
+The `grafana-server` binary .tar.gz needs the working directory to be the root install directory where the binary and the `public` folder are located.
+
+To start the Grafana server, run the following command:
+
+```bash
+./bin/grafana-server web
```

Comment:
I'm not sure where this command came from, but it's incorrect.  Passing `web` as a parameter doesn't do anything.
```suggestion
./bin/grafana-server
```